pkiinit/README.md

```diff
@@ -16,3 +16,11 @@ For running in Docker, please build the binaries and docker images before run `d
 ## Run Docker
 
 On the command line console, run `docker-compose up --build` from the directory `pkiinit` to start the whole Docker container stack.
+
+## Command Line Options
+
+1. --generate: to genearte new TLS certificates
```

Comment:
Small but should these be numbered?  wouldn't "--property ..." be easier to understand?  you don't pass in numbers to the binary you just pass in the flag.

Reply:
Agree. I'll remove the number.

Reply:
number usually implied the priority or importance so that is a good point. :-)